helm-chart/templates/04-hub-deployment.yaml

```diff
@@ -28,6 +28,8 @@ spec:
           command:
             - ./hub
             {{ .Values.tap.debug | ternary "- -debug" "" }}
+            - e
+            - TELEMETRY_DISABLED={{ .Values.tap.notelemetry }}
```

Comment:
```suggestion
```

Add it to here instead:

https://github.com/kubeshark/kubeshark/blob/bf182b633041ebcb975728f41bce86f214328e8f/helm-chart/templates/13-config-map.yaml#L10-L16